Formula/simgrid.rb

```diff
@@ -1,20 +1,11 @@
 class Simgrid < Formula
   desc "Studies behavior of large-scale distributed systems"
   homepage "http://simgrid.gforge.inria.fr"
-  url "https://gforge.inria.fr/frs/download.php/file/33686/SimGrid-3.11.1.tar.gz"
-  sha256 "7796ef6d4288462fdabdf5696c453ea6aabc433a813a384db2950ae26eff7956"
-
-  bottle do
```

Comment:
The bottle block should not be changed or removed. It's updated programmatically.

Reply:
Oh. Alright. Will fix that.